books/free-programming-books-langs.md

```diff
@@ -1595,6 +1595,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Developing Applications With Objective Caml](http://caml.inria.fr/pub/docs/oreilly-book/)
 * [Functional Programming in OCaml](https://www.cs.cornell.edu/courses/cs3110/2019sp/textbook/) - Michael R. Clarkson
 * [OCaml From the Ground Up](https://ocamlbook.org) - Daniil Baturin (HTML) (:construction: *in process*)
+* [OCaml from the Very Beginning](https://johnwhitington.net/ocamlfromtheverybeginning/) - John Whitington (HTML, [PDF](https://johnwhitington.net/ocamlfromtheverybeginning/mlbook.pdf), [GitHub Repo](https://github.com/johnwhitington/mlbook))
```

Comment:
Just one link, please

Reply:
Thanks for the review, @eshellman! Should I keep the GitHub repo link?